Add support for pool size

panthershark opened the pull request:
This pull requests adds the option to create more than one connection per host in a pool.  

What test runnner are you using? I tested it in a few of my applications that rely on node-cassandra-cql and it seems to work great.  I was not able to determine which test runner the project was designed for.  If you tell me, then I can ensure this does not break any tests (and write one for the connection pool counts).

Great library.  Thanks for maintaining it.

Commit:
added support for connection pool size allowing multiple connections per host

panthershark commented:
duh.. just saw that it is using node-unit.  I can add the connection count test over the weekend.

jorgebay commented:
:+1:
I tested the changes locally and it does not affect the other tests :)

It would be good to have the loop the other way around, to get an array of connections like this:
`[host1, host2, ..., hostn, host1, host2, ..., hostn, ...]`
In this way, the methods to get the next connection (specially for prepared statements) will always get a different host.

If you change how the loops are declared and create a small unit test I will merge it to the master.

Cheers!
J

panthershark commented:
Rad. I plan to get to this next week. I am going out of town this weekend.

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
flipped the order of the connection loops

panthershark commented:
Updated the loops.  I added the test script to package.json and you can now run via npm test.  

However, the tests seem to hang and I am not sure if this is something with my env or the module.  Do I need to setup the unittestkp1_2 keyspace before running?  Is there something else needed?

I was able to test the module with my apps that use it and it seems to work.  

Cheers!

Commit:
remove console.log which was leftover from debugging

jorgebay commented:
It looks good, I'll merge it tomorrow!

About the tests hanging, it is indeed weird... [travisci](https://travis-ci.org/jorgebay/node-cassandra-cql/builds/11469064) says it's all good: I'll look into it.

panthershark commented:
Thanks.  Let me know if this needs anything else.

jorgebay commented:
Done!
I added you to package contributors
